wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/WrappersToGenerate.kt

```diff
@@ -306,5 +306,120 @@ val wrappersToGenerate = listOf(
             "log-level" to EnumTyping("LogLevel", listOf("minimal", "standard", "verbose")),
             "security" to EnumTyping("Security", listOf("strict", "loose")),
         ),
-    )
+    ),
+    WrapperRequest(
+        ActionCoords("actions", "create-release", "v1"),
+        mapOf(
+            "draft" to BooleanTyping,
+            "prerelease" to BooleanTyping,
+        )
+    ),
+    WrapperRequest(
+        ActionCoords("actions", "first-interaction", "v1"),
+    ),
+    WrapperRequest(
+        ActionCoords("actions", "github-script", "v6"),
+        mapOf(
+            "debug" to BooleanTyping,
+            "previews" to ListOfTypings(","),
+            "result-encoding" to EnumTyping("Encoding", listOf("string", "json"))
+        )
+    ),
+    WrapperRequest(
+        ActionCoords("actions", "labeler", "v3"),
```

Comment:
There's already [v4](https://github.com/actions/labeler/releases/tag/v4):
```suggestion
        ActionCoords("actions", "labeler", "v4"),
```

Reply:
done